src/meta/processors/admin/InstallPluginProcessor.cpp

```diff
@@ -0,0 +1,67 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+
+#include "meta/processors/admin/InstallPluginProcessor.h"
+#include "common/fs/FileUtils.h"
+
+namespace nebula {
+namespace meta {
+
+// When installing, just register it, do not always open it, open it when using
+void InstallPluginProcessor::process(const cpp2::InstallPluginReq& req) {
+    const auto& pluginItem = req.get_item();
+    auto& pluginName = pluginItem.get_plugin_name();
+    auto& soName = pluginItem.get_so_name();
+    folly::SharedMutex::WriteHolder wHolder(LockUtils::pluginLock());
+
```

Comment:
A question? using so file in graph layer, and register so file in meta layer. Therefore, how to ensure that the file exists in the graph server? because some times maybe graphd and metad on different systems.

Reply:
good question.

At present, there are so files in the corresponding directories of all machines, include graphd and metad.